bindings/python/fdb/six.py

```diff
@@ -38,15 +38,15 @@
 PY34 = sys.version_info[0:2] >= (3, 4)
 
 if PY3:
-    string_types = str,
-    integer_types = int,
-    class_types = type,
+    string_types = (str,)
```

Comment:
I believe our usual policy with files that come from external sources is not to apply any of our own styles to them. That makes it easier to compare and pull in changes that happen externally.

Is it easy to exclude such files from consideration?